src/aks-preview/azext_aks_preview/_help.py

```diff
@@ -260,9 +260,9 @@
         - name: --appgw-name
           type: string
           short-summary: Name of the application gateway to create/use in the node resource group. Use with ingress-azure addon.
-        - name: --appgw-subnet-prefix
+        - name: --appgw-subnet-cidr
```

Comment:
So this is a breaking change to existing users, right ?

Reply:
yes, this is a breaking change for existing users of addon.